lib/models/stop.rb

```diff
@@ -6,7 +6,8 @@ class Stop < ActiveRecord::Base
   has_many :departures
   has_many :issues
 
-  validates :name, :hastus_id, presence: true, uniqueness: true
+  validates :name, presence: true
```

Comment:
As far as I can tell, stop names are not unique in the GTFS feed (at least currently):

```
$awk -F, '{print $3}' gtfs_files/stops.txt | sort | uniq -c | sed -ne '/^ \+2/p'
      2 "Beech / Hospital"
      2 "Broadway / Pine"
      2 "Broadway / Theodore"
      2 "Broadway / Western"
      2 "Cabot / Bond"
      2 "Cabot / Bridge"
      2 "Cabot / Canal"
      2 "Cabot / Clemente"
      2 "Carew and Main(Tripper Timepoint)"
      2 "Central / Spruce"
      2 "Chicopee / Baptist"
      2 "Dickinson / Trenton"
      2 "Grove / Front"
      2 "Highland / Prospect"
      2 "Hillside / Westfield"
      2 "King / Logan"
      2 "Liberty / Carew"
      2 "Liberty / Cleveland"
      2 "Locust / Darwell"
      2 "Longmeadow/Summit (Flag Stop)"
      2 "Main and Sargent(Holyoke Tripper)"
      2 "Main / Bancroft"
      2 "Main / Bridge"
      2 "Main / Canterbury"
      2 "Main / George"
      2 "Main / Meadow"
      2 "Main / William"
      2 "Mason Square"
      2 "Memorial / Main"
      2 "Russell / Elmwood"
      2 "Springfield / Chapin"
      2 "State and St. James(Tripper Timepoint)"
      2 "State / Walnut"
      2 "Westfield / Bray Park"
      2 "Westfield / Connecticut"
      2 "Westfield / Woodland"
      2 "Whitney / Unnamed Street"
      2 "Worthington / Federal"
```